sender.go

```diff
@@ -333,7 +334,7 @@ func (s *LineSender) Int64Column(name string, val int64) *LineSender {
 // Column name cannot contain any of the following characters:
 // '\n', '\r', '?', '.', ',', ”', '"', '\\', '/', ':', ')', '(', '+',
 // '-', '*' '%%', '~', or a non-printable char.
-func (s *LineSender) Long256Column(name, val string) *LineSender {
```

Comment:
The documentation should be simplified. We no longer need to explain the string format. Instead, we should explain validation rules: only non-negative numbers that fit into 256-bit unsigned integer are supported and any other input value would lead to an error.

Reply:
Okay. Another thing

Since there are multiple ways to set `big.Int` value, there goes a confusion of setting a value with `0x` or not.
Example:
```go
	var bigVal big.Int
	// Create a big int number from string
	bigVal.SetString("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF", 16)
	// Create a big int number from int in decimal
	bigVal.SetInt64(1000000000000000000)
	// Create a big int number from int in hexdecimal
	bigVal.SetInt64(0x1234)
	// Create a big int number from bytes
	bigVal.SetBytes([]byte{0x01, 0x02, 0x03, 0x04, 0x05})
```
Is there something we can do about it?
I suggest adding examples for the same.

Reply:
It's up to the user how to initialize a `big.Int` instance. They should refer to the official `math/big` package docs, not ours.